pages/common/gcloud.md

```diff
@@ -1,13 +1,14 @@
 # gcloud
 
 > The official CLI tool for Google Cloud Platform.
+> Some subcommands such as `gcloud iam` have their own usage documentation.
```

Comment:
The eventual aim is to document all subcommands. Can we reword this to be more generic?

Reply:
Sure

```suggestion
> Note: `gcloud` subcommands have their own usage documentation.
```